### PR TITLE
Add a new script and service which copies only the run meta data

### PR DESCRIPTION
**Pull request by fschlueter**
@cozzyd very simple ...

### CODE REVIEW

**Comment by cozzyd** [comment]
Do you need to update install target? (Maybe it copies everything already, I don't remember)

**Comment by fschlueter** [comment]
what do you mean?

**Comment by cozzyd** [comment]
it copies everything already in the Makefile (so no need to add! sorry!): 
https://github.com/RNO-G/rno-g-ice-software/blob/master/Makefile#L90

File: scripts/rno-g-simple-copy [hunk 1]
```diff
@@ -53,7 +53,8 @@ do
   FREESPACE=`df /data | awk 'NR>1 {print $4}'`
   if [ $FREESPACE -lt $THRESH1 ] ;
   then
-    find /data/daq -${CMP} +${AGE1} -delete
+    # Try to copy a last time and delete if / after copied
+    find /data/daq -${CMP} +${AGE1} | rsync  -avh --remove-source-files --files-from=- /data/daq 10.1.0.1:/data/ingress/station$STN/
```
Comment (cozzyd) [inline scripts/rno-g-simple-copy]: have you checked that this rsync command does what we want? I'm not 100% sure how --files-from treats relative paths

Comment (cozzyd) [reply scripts/rno-g-simple-copy]: May be worth trying a test command with --dry-run if you haven't already

Comment (fschlueter) [reply scripts/rno-g-simple-copy]: okay I had to slightly change the command to
```
find /data/daq -mtime +5 | rsync  -navh --remove-source-files --files-from=-  / 10.1.0.1:/data/ingress/station$STN/
```

Last few lines:
```
data/daq/run240315/header/000067.hd.dat.gz
data/daq/run240315/header/000142.hd.dat.gz
data/daq/run240315/waveforms/
data/daq/run240315/waveforms/000000.wf.dat.gz
data/daq/run240315/waveforms/000067.wf.dat.gz
data/daq/run240315/waveforms/000142.wf.dat.gz

sent 42.01K bytes  received 3.91K bytes  10.20K bytes/sec
total size is 1.47G  speedup is 31,952.13 (DRY RUN)
```

**Comment by fschlueter** [comment]
can we go ahead with this?

**Comment by cozzyd** [comment]
What happens if the two independent rsync processes copy the same file at once?

**Comment by fschlueter** [comment]
I have no idea. Do you want to exclude the meta data from the other copy script?

**Comment by cozzyd** [comment]
That might make more sense, yeah, though I'm still not sure what happens with intermediate directory creation, though that's probably not as big of an issue as random .tmp potentially files being left around.

**Comment by fschlueter** [comment]
@cozzyd ping

**Comment by cozzyd** [comment]
alright, since directory creation should be atomic, I think this should be fine. Do you wanna go ahead and test it on a station to make sure the files all get copied?

**Comment by fschlueter** [comment]
okay, quite some changes. have a look if you want @cozzyd. The services are now running of st11

**Comment by fschlueter** [comment]
the `run240682/aux/acq-file-list.txt` is frequently updated. And since we run a `rsync` command every minute at the moment the newest version of `run240682/aux/acq-file-list.txt` is copied every minute. Should we increase that to 2-5 min?

**Comment by cozzyd** [comment]
So that file list is currently used for rootification (to see if new data is there).  It should be kept with the data I think.

**Comment by cozzyd** [comment]
It should be possible to avoid the cd with the right flags to find, I think ? But the cd might be simpler.

**Comment by fschlueter** [comment]
I don’t want to delete or skip the file, I was just wondering if we should increase the sleep from 60 to something more

**Comment by cozzyd** [comment]
well, 60 seconds is nice in cases when you're trying to quickly look at a run in progress with minimal latency, but that's a small minority of the time. Can probably increase that on both scripts... (or try to read it from a file or something, but that's not so much fun to do in a shell script).

**Comment by fschlueter** [comment]
Okay, I will merge this PR since the test on station 11 seems successful. I can manually start the new service on stations with running daq. Is there any problem with running `make service-install` while the daq is running? On station11 I got some system msg on the journalctl output...

**Pull request merged**